sympy/tensor/array/expressions/conv_array_to_matrix.py

```diff
@@ -283,23 +283,23 @@ def _(expr: PermuteDims):
             p2 = permuted[2*i+1]
             if p1 // 2 != p2 // 2:
                 return _permute_dims(mat_mul_lines, permutation)
-            pos = p1 // 2
+            posi = p1 // 2
```

Comment:
Why store this in a variable at all, it gets used only once due to the if-else (unless it gets moved up)

Reply:
I'm not sure I understand, It gets used in either of two branches below.

Reply:
Right, it gets used in either branch, so only once. It's not exactly a complicated expression, so why not just put `p1 // 2`` directly where `posi` is used?

Reply:
There can be good reasons for naming something even if it is only used once but since this isn't a particularly good name anyway I'll do that.